modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -14,18 +14,26 @@ settings and run `openshift-install` to install a cluster and `oc` to work with
 for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques]
 for a presentation on Tang.
 
-. Add kernel arguments to configure networking when you do the RHCOS installations for your cluster.
-If you miss this step, the second boot will fail.
-For example, to configure DHCP networking, identify `ip=dhcp`
-or set static networking when you add parameters to the kernel command line.
+. To configure networking when you do the {op-system-first} installations for your cluster, do one of the following:
```

Comment:
Where are these `ip=` kernel arguments to be specified? I am unsure from this step as to exactly where to include the `ip=` kernel arguments. If they are intended to be in the `kernelArguments:` section of the `./99-openshift-worker-tang-encryption.yaml` or `./99-openshift-master-encryption.yaml` files, then I think this step should come after step 7 or 8, where those files are first mentioned. If they are meant to be included somewhere else, then I think that this step should state where.

Reply:
I think this strayed away from the original intention once I added the `rd.neednet=1`, which was to instruct users to add kargs when setting up networking. `DHCP` and `static networking` were listed examples. Because we've now moved `rd.neednet=1` to later in the module, I've restored the original wording of this step.

Reply:
That looks good to me, thanks Bob.